src/OpenTelemetry.AutoInstrumentation.Native/util.cpp

```diff
@@ -97,6 +99,53 @@ std::vector<WSTRING> GetEnvironmentValues(const WSTRING& name)
     return GetEnvironmentValues(name, L',');
 }
 
+
+std::vector<WSTRING> GetEnvironmentVariables(const std::vector<WSTRING> &prefixes)
+{
+    std::vector<WSTRING> env_strings;
+#ifdef _WIN32
+    auto env_variables = GetEnvironmentStrings();
+    int prev = 0;
+    for (int i = 0;; i++) {
+        if (env_variables[i] == '\0')
```

Comment:
nit: I would revert this to reduce nesting.

Reply:
Fixed: 9f1d8300e51eda9af97d7eb928e44a5f815a3bd5